[HW]1072956

brianlin314 opened the pull request:
在ex6完成所有ppt要求後，在github中要選擇branch，因為每次都會回到ex0，所以要提醒同學記得回到ex6的branch，切記不要交到ex6-2或是ex0，最後在pull request
Signed-off-by: 1072956 <s1072956@mail.ncyu.edu.tw>

jrjang commented:
Hi @brianlin314 ,

1. commit message 格式不對
  1.1 commit summary 是 [HW] 學號
  1.2 以下描述是要在 commit message 中
```
在ex6完成所有ppt要求後，在github中要選擇branch，因為每次都會回到ex0，所以要提醒同學記得回到ex6的branch，切記不要交到ex6-2或是ex0，最後在pull request
```

jrjang commented on README.md:
```diff
@@ -3,6 +3,7 @@ Example 6:
 1. 依照投影片描述做修改
 
 2. 完成後, 在 ncyu-2021-ex6 資料夾下, 執行以下指令
+因為ex6在完成所有ppt要求後，在github中要選擇branch，因為每次都會回到ex0，所以要提醒同學季的回到ex6的branch，切記不要交到ex6-2，最後在pull request
```
麻煩放在第 3 點, 並重新改寫描述 (以學生角度)

Commit:
[HW] 1072956

要解決之問題:
在練習ex6並完成所有ppt的要求後，在github中要選擇ex6的branch並pull request，但每次fork之後都要重新選擇branch，如果忘記了，便會停留在ex0的branch。
解決方法:
提醒同學們記得先回到ex6的branch再進行pull request;另外，在做完ex6的所有要求後，會產生ex6-2的branch，也請同學記得是到ex6的branch進行pull request。

Signed-off-by: 1072956 <s1072956@mail.ncyu.edu.tw>
